app/views/content_items/field_of_operation.html.erb

```diff
@@ -16,11 +16,13 @@
   </div>
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-third">
-      <%= render "govuk_publishing_components/components/contents_list", {
-        contents: @content_item.contents
-      } %>
-    </div>
+    <% if @content_item.contents.present? %>
```

Comment:
The commit message is a bit blank for this. Does it make sense to have this as a separate commit, or should we squash into the previous commit?

Reply:
have updated